model/lib/extensions/presentation_mailbox_extension.dart

```diff
@@ -49,6 +49,8 @@ extension PresentationMailboxExtension on PresentationMailbox {
 
   bool get isSubscribedMailbox => isSubscribed != null && isSubscribed?.value == true;
 
+  bool get isSubaddressingAllowed => rights != null && rights?['anyone']?.contains('p') == true;
```

Comment:
And keywords (`anyone `, `p`) should be defined as `const`

Reply:
should i define those consts in a common location for both `presentation_mailbox_extension` and `mailbox_api` ?

Reply:
IMO, we should do so